src/core/dbus-cgroup.c

```diff
@@ -826,6 +829,31 @@ int bus_cgroup_set_property(
 
                 return 1;
 
+        } else if (STR_IN_SET(name, "MemoryLow", "MemoryHigh", "MemoryMax")) {
```

Comment:
> MemoryMax and MemoryLimit are equivalent and translated back and forth. The legacy path uses MemoryMax if MemoryLimit is not set.

Indeed. This works. But `systemctl show --property MemoryLimit` shows incorrect value:
```sh
# MemoryLimit=100M
-bash-4.3# systemd-run --property MemoryLimit=100M -r sh -c 'cat </dev/zero >/dev/null'
Running as unit: run-rda141fcfab2b41e388a34a4578c8c0f5.service

-bash-4.3# cat /sys/fs/cgroup/memory/system.slice/run-rda141fcfab2b41e388a34a4578c8c0f5.service/memory.limit_in_bytes
104857600

-bash-4.3# systemctl show --property MemoryLimit --value run-rda141fcfab2b41e388a34a4578c8c0f5.service --no-pager
104857600
```

```
# MemoryMax=100M
-bash-4.3# systemd-run --property MemoryMax=100M -r sh -c 'cat </dev/zero >/dev/null'
Running as unit: run-r91d650df2dab4051abeaa97ebbc27e97.service

-bash-4.3# cat /sys/fs/cgroup/memory/system.slice/run-r91d650df2dab4051abeaa97ebbc27e97.service/memory.limit_in_bytes
104857600

-bash-4.3# systemctl show --property MemoryLimit --value run-r91d650df2dab4051abeaa97ebbc27e97.service --no-pager
18446744073709551615
```